getting_started/audio.rst

```diff
@@ -14,45 +15,46 @@ All of this is supported by Escoria.
 Playing music and sound effects
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Escoria features three built-in audio players, which are accessible by their
+Escoria features three built-in audio players which are accessible by their
 ESC global IDs:
 
-- _music: Used for looping background music
-- _sound: Used for playing sound files
-- _speech: Used for voice support
+- `_music`: Used for looping background music
+- `_sound`: Used for playing sound files
+- `_speech`: Used for voice support
 
-Playing music and sound effects is done by using the
+Playing audio is done by using the
 :doc:`play_snd </api/PlaySndCommand>` ESC command. It expects the path to the
-sound effect or music ressource and the global id of the audio player.
+specific audio resource to be played and the global ID of the desired audio
+player.
 
-If the audio is unspecified, it defaults to ``_sound``.
+If the audio player is unspecified, it defaults to ``_sound``.
 
 Examples
 --------
 
 - ``play_snd res://music/happy.ogg _music``
 
-  Plays the background music ``res://music/happy.ogg``
+    Plays the background music ``res://music/happy.ogg``
```

Comment:
What were you aiming for with this and the next alignment because they don't match. I'd align them like before on the first character of the list item line.

Reply:
Think that's just an accidental hit of the space bar. Will correct.